src/yaml-support/tkn-yaml-scheme-generator.ts

```diff
@@ -83,6 +112,63 @@ function injectMarkdownDescription(templateObj: any): {} {
   return templateObj;
 }
 
+function injectVariables(templateObj: any, docs: TknDocument[], tasks: TknTask[]): unknown {
```

Comment:
can you remove `any` type and use the specific type which reflects the templateObj.

Reply:
I can, but it would be difficult, as `templateObj` it is object created from https://github.com/redhat-developer/vscode-tekton/blob/master/scheme/tekton.dev/v1beta1_Pipeline.json